### PR TITLE
Update README.md to add SuperTokens - open core self hosted auth provider

### PR DESCRIPTION
**Pull request by rishabhpoddar**
Thank you for taking the time to work on a PR for Awesome-Sysadmin!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software)
- [x] Software your are submitting is not your own, unless you have a healthy ecosystem with a few contributors (which aren't your sock puppet accounts) - we have a community of 1k+ devs on [our Discord](https://supertokens.com/discord).
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 80 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted)
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-foss/awesome-sysadmin/issues) or [PRs](https://github.com/awesome-foss/awesome-sysadmin/pulls), including closed ones.
- [ ] Any category you are creating has the minimum requirement of 3 items.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

--------------

Please take some time to answer the following questions as best you can:

- **Why is it awesome?**

Because it is unique in the following ways:
- Has end to end session management with HttpOnly cookies support
- Modules are decoupled so that users can use any part of our software without using all of it. For example, use our session management module without using our login module. (or vice versa)
- Vey customisable - all customisations to the auth business logic can be done in the user's backend layer without touch the java code.

- **Have you used it? For how long?**

Been using it for 3 years.

- **Is this in a personal or professional setup?**

Professional setup

- **How many devices/users/services/... do you manage with it?**

We manage over 1000 users with is on our site (supertokens.com), but we also have over 1k+ companyes / users using supertokens for their websites.

- **Biggest pros/cons compared to other solutions?**

Biggest pro is the level of customisability it offers. Biggest con is that it requires you to use a backend for which we have an SDK for (golang, python or nodejs)

- **Any other comments about your use case, things you've found excellent, limitations you've encountered... ?**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -417,6 +417,7 @@ _Related: [DNS - Servers](#dns---servers)_
 - [Authelia](https://www.authelia.com/) - The Single Sign-On Multi-Factor portal for web apps. ([Source Code](https://github.com/authelia/authelia)) `Apache-2.0` `Go`
 - [Authentik](https://goauthentik.io/) - Flexible identity provider with support for different protocols. (OAuth 2.0, SAML, LDAP and Radius). ([Source Code](https://github.com/goauthentik/authentik)) `MIT` `Python`
 - [KeyCloak](https://www.keycloak.org) - Open Source Identity and Access Management. ([Source Code](https://github.com/keycloak/keycloak)) `Apache-2.0` `Java`
+- [SuperTokens](https://supertokens.com) - Open Core Identity, Access and Session Management. ([Source Code](https://github.com/supertokens/supertokens-core)) `Apache-2.0` `Java`, `NodeJS`, `Python`, `Golang`
```
Comment (KieranRobson) [inline README.md:420]: > Sentence case

> https://github.com/supertokens/supertokens-core#features---click-here-to-see-the-demo-app

> ✅ If you require any modifications to the auth APIs, those would need to be done on the backend SDK level (for example Node, Golang, Python..). So you’d rarely need to directly modify/work with the Java code in this repo.

```suggestion
- [SuperTokens](https://supertokens.com) - Add secure login and session management to your apps. ([Demo](https://thirdpartyemailpassword.demo.supertokens.com/auth/?redirectToPath=), [Source Code](https://github.com/supertokens/supertokens-core)) `Apache-2.0` `Java`
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Couple changes suggested

**Comment by rishabhpoddar** [comment]
Thanks @KieranRobson - accepted the changes :)

**Comment by nodiscc** [comment]
https://github.com/supertokens/supertokens-core/blob/master/ee/LICENSE.md:
> This software and associated documentation files (the "Software") may only be used in production, if [multiple conditions]
> it is forbidden to copy, merge, publish, distribute, sublicense, and/or sell the Software.

https://opensource.org/osd:
> The license must not restrict anyone from making use of the program in a specific field of endeavor
> The license shall not restrict any party from selling or giving away the software

**Pull request closed without merging**